core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/HDFSFileDeleteAction.java

```diff
@@ -91,16 +110,17 @@ public boolean accept(Path path) {
         LOG.warn("Removal of {} was unsuccessful.", path.toString());
       }
     }
+
   }
 
   @Override
   public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
     if (config.fileRegex != null) {
       try {
-        Pattern.compile(config.fileRegex);
+        Pattern filter = Pattern.compile(config.fileRegex);
```

Comment:
why add this back? this should generate a warning for an unused variable. Just do Pattern.compile